backend/models.py

```diff
@@ -0,0 +1,23 @@
+from pydantic import BaseModel
+
+from datetime import datetime
+
+from typing import List
+
+
+class Task(BaseModel):
+    """Class for tasks."""
```

Comment:
It's okay to have this documentation now, but we should create another task/spike to normalize how we will write the functions/classes documentation.

Reply:
So this is a future task to achieve?

Reply:
That's right, we will need to plan an create a new task in trello